plugins/modules/scc_product.py

```diff
@@ -58,12 +61,13 @@
 
 
 from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloAnsibleModule
+from distutils.version import LooseVersion
 
 
 def main():
     module = KatelloAnsibleModule(
         foreman_spec=dict(
-            scc_product=dict(required=True, type='entity', aliases=['friendly_name'], scope=['scc_account'], thin=False),
+            scc_product=dict(required=True, type='entity', scope=['scc_account'], thin=False),
```

Comment:
CI complains about this now, as you allow the `friendly_name` alias in the documentation, but not in the spec.

Please revert this line to read
```python
scc_product=dict(required=True, type='entity', aliases=['friendly_name'], scope=['scc_account'], thin=False),
```
and we should be good to go!

Reply:
Fixed!